### ISSUE TITLE
Changing the format of the tool.

### ISSUE THREAD

**Issue by KedarKshatriya** [ideonate/jupyter-innotater#20]
Is there any option to format the view of the tool. Like to shift the label boxes below the image viewer. Since using the default format gives very little space for the image.

**Comment by danlester** [comment]
Thank you for the suggestion!

I think it would be useful at least to have a configuration option for inputs / targets to be vertically rather than horizontally stacked.

It would also be fantastic to have a 'full screen' pop out button!

There isn't currently any funding assigned for such work on this software, but I will take a quick look if I get a chance.

**Comment by eric-tramel** [comment]
@danlester I'd use this feature :) I find myself running annotation on some very large-format images, and so one runs into problems navigating the image itself with a wide-sidebar for the multi-class bounding box use-case.

As a side note: if one wants to get a bit more bang-for-buck out of Jupyter on such images, try out the following in the first cell to reduce the significant side margins ([from here](https://stackoverflow.com/questions/21971449/how-do-i-increase-the-cell-width-of-the-jupyter-ipython-notebook-in-my-browser)).

```python
from IPython.core.display import display, HTML
display(HTML("<style>.container { width:100% !important; }</style>"))
```

**Comment by danlester** [comment]
Thanks for the snippet - really helpful!

**Comment by danlester** [comment]
I have released version 0.2.1 of jupyter-innotater which adds a `vertical` flag to `Innotater`.

So you can invoke `Innotater(... , vertical=True)` to have inputs/targets stacked vertically instead of horizontally.

I hope this helps, although ideally we would also have a full screen pop-out button.

I'll close for now.

**Comment by KedarKshatriya** [comment]
Thank you so much.
